Algorithms/Search/DFS.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+
+namespace DFS
+{
+    /// <summary>
+    /// Depth First Search.
+    /// </summary>
+
+    public class DepthFirst<T>
+    {
+        // Returns true if item exists.
+
+        public bool Find(IGraph<T> graph, T vertex)
+        {
+            return dfs(graph.ReferenceVertex, new HashSet<T>(), vertex);
+        }
+
+        // Recursive DFS.
+
+        private bool dfs(IGraphVertex<T> current,
+            HashSet<T> visited, T searchVetex)
+        {
+            visited.Add(current.Key);
+
+            if (current.Key.Equals(searchVetex))
+            	return true;

```

Comment:
Codacy found an issue: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6325512)